docs/installation.rst

```diff
@@ -113,6 +113,11 @@ Standard Installation
 
     helm install robusta robusta/robusta -f ./generated_values.yaml --set clusterName=<YOUR_CLUSTER_NAME> # --set isSmallCluster=true
 
+    
+.. note::
+
+      You can provide configuration values as Kubernete secrets. See `Configuration secrets <https://docs.robusta.dev/master/user-guide/configuration-secrets.html>`_ guide.
```

Comment:
I would change to:
Sensitive configuration values can be stored in Kubernetes secrets. See `Configuration secrets <https://docs.robusta.dev/master/user-guide/configuration-secrets.html>`_ guide.

Reply:
Fixed.